boa/src/builtins/number/mod.rs

```diff
@@ -83,23 +79,36 @@ impl BuiltIn for Number {
         .static_method(Self::number_is_integer, "isInteger", 1)
         .build();
 
-        let global = context.global_object();
-        make_builtin_fn(
-            Self::parse_int,
-            "parseInt",
-            &global,
-            PARSE_INT_MAX_ARG_COUNT,
-            context,
-        );
-        make_builtin_fn(
-            Self::parse_float,
-            "parseFloat",
-            &global,
-            PARSE_FLOAT_MAX_ARG_COUNT,
-            context,
-        );
-        make_builtin_fn(Self::global_is_finite, "isFinite", &global, 1, context);
-        make_builtin_fn(Self::global_is_nan, "isNaN", &global, 1, context);
+        fn build_in_function(
```

Comment:
Shouldn't this be named `built_in_fn()`? Also, clippy is not happy about having the function definition after some statements.

In any case, if `make_builtin_fn()` is not being used, can it be replaced and have this function shared between all the different global objects?

Reply:
This was the only place where `make_builtin_fn()` is used on the global object. In some other places we use it for other objects. It does not work anymore for the global object, because it uses the `insert_property` helper method, that inserts on the `properties` of the object. These properties are not used for the global object anymore.

I think, I will modify or add another function like `Context::register_global_function` for this. I think the only problem with that one was, that it added the global function as a constructor.

Reply:
I decided to go with `Context::register_global_builtin_function`.